tests/lib/common.py

```diff
@@ -389,3 +392,64 @@ def configure_postfix(device):
     device.expect(device.prompt)
     device.sendline("service postfix reload")
     assert 0 != device.expect(['failed']+ device.prompt, timeout = 20) , "Unable to reolad server with new configurations"
+
+
+def file_open(file_path):
+    extension = os.path.splitext(file_path)[1]
+    with open(file_path) as f:
+        try:
+            if "json" in extension:
+                read_data = json.load(f)
+            elif "txt" in extension:
+                read_data = f.read()
+            return read_data
+        except Exception as e:
+            print "Could not read file from the path:", file_path
+            raise (e)
```

Comment:
Can we keep this a bit simplier for now:
```
def file_open_json(file):
    with open(file) as f:
        return json.load(f)
```

We should throw an exception if something fails, since we're only loading a json file.

Also, this could and should be in it's own commit.

Reply:
Agreed will change the same